kivy/input/motionevent.py

```diff
@@ -484,3 +484,21 @@ def is_mouse_scrolling(self, *args):
         .. versionadded:: 1.6.0
         '''
         return 'button' in self.profile and 'scroll' in self.button
+
+
+class HoverEvent(MotionEvent):
```

Comment:
Don't forget the documentation here, with the ..versionadded tag (look around the code to find out)

Reply:
I'll add documentation at the end development, once we iron-out this feature.